examples/basic.c

```diff
@@ -9,6 +9,7 @@ int main(int argc, char *argv[]) {
     exit(1);
   }
 
+  mg_init();
```

Comment:
using C-APi with C++-API looks awkward. Maybe have static wrapper methods in Client?

Reply:
Yep, I agree, will do it to be more consistent.